runtime/doc/api.txt

```diff
@@ -2685,6 +2685,10 @@ nvim_buf_set_extmark({buffer}, {ns_id}, {line}, {col}, {opts})
                     • "overlay": display over the specified column, without
                       shifting the underlying text.
                     • "right_align": display right aligned in the window.
+                    • "eol_right_align": display right aligned in the window
+                      unless the virtual text is longer than the space
+                      available. If the virtual text is too long, it is
+                      truncated to fit in the window after the EOL character.
```

Comment:
This doesn't make it clear that when the line is wrapped, the virtual text is shown after the end of the line, not on a previous screen line.

Reply:
I added on a sentence for clarification on this subject.